test/libsolidity/syntaxTests/inheritance/interface/multiple_parents.sol

```diff
@@ -0,0 +1,12 @@
+interface SuperA {
+    function test() external returns (uint256);
+    function testA() external returns (int128);
+}
+
+interface SuperB {
+    function test() external returns (uint256);
+    function testB() external returns (int256);
+}
+
+interface Sub is SuperA, SuperB {
+}
```

Comment:
We should require an override here. This is to explicitly state that the functions inherited from the different interfaces are in fact compatible.

Reply:
Oh, I thought having no error here was what you were asking for at https://github.com/ethereum/solidity/pull/8091#discussion_r368502360. I'll revert that change.

Reply:
Ah sorry, what I meant was if the smart contract can be changed so that the error disappears :)

Reply:
It can be, but that's tested elsewhere in `overrides_multiple.sol`, function `test5`.